src/vat/remotes.js

```diff
@@ -181,19 +180,22 @@ export function makeDecisionList(
       return;
     }
 
-    for (const d in decisionList) {
+    const doDecisionsAgree = decisionList.every(d => {
```

Comment:
Yep, looks like the same logic, but now that we've turned off the lint about for..of, can we remove this change? If we do want to include it, I think I'd flip the polarity: use `decisionList.some` instead of `every`, name it `equivocated` instead of `doDecisionsAgree`, return `true` in the one case where we log and `false` otherwise, and change the `if (!doDecisionsAgree)` to `if (equivocated)`. I'm on the fence about which one (for..of or `equivocated=`) is more legible.. I'll leave that to your judgement.

Reply:
Yup, we can remove this change